tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -62,12 +62,12 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_ECDSA_C:MBEDTLS_ECDSA_DETERMINISTIC:MBEDTLS_ECP_DP_SECP256R1_ENABLED:MBEDTLS_SHA256_C */
```

Comment:
Pre-existing, but topical: since verification for ECDSA is the same whether the signature has been generated deterministically or not, can we remove the dependency on deterministic ECDSA here, and set the verifying algorithm to 'regular' ECDSA?